storm-core/src/clj/org/apache/storm/daemon/executor.clj

```diff
@@ -302,12 +303,13 @@
         ]
     (disruptor/consume-loop*
       (:batch-transfer-queue executor-data)
-      (disruptor/handler [o seq-id batch-end?]
-        (let [^ArrayList alist (.getObject cached-emit)]
-          (.add alist o)
-          (when batch-end?
-            (worker-transfer-fn serializer alist)
-            (.setObject cached-emit (ArrayList.)))))
+      (DisruptorUtils/clojureHandler
```

Comment:
Same here, lest just inline the implementation instead of translating something to java that depends on clojure.